src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  font-family: "aventir", sans-serif;
```

Your font doesn't work because of this
```suggestion
  font-family: "Avenir", sans-serif;
```